app/jobs/run_consistency_checks_job.rb

```diff
@@ -4,6 +4,10 @@ class RunConsistencyChecksJob < ApplicationJob
   queue_as :default
 
   def perform(*_args)
+    return unless FeatureService.enabled?("run_consistency_check_job")
```

Comment:
```suggestion
return unless (FeatureService.enabled?("run_consistency_check_job") && ConsistencyCheck.any?)
```

i think you want both of these cases to be true for this job to run so think you could simplify it to the above?

Reply:
Nice, have redacted it into single unless 👍